graph/ForceDirectedGraph/ExampleForceDirectedGraph.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Graph from '../Graph';
+import ForceDirectedGraph from './ForceDirectedGraph';
+
+const ExampleForceDirectedGraph = () => {
+    const graph = new Graph(['0', '1', '2', '3', '4', '5', '6'], ['0', '2', '3', '3', '3', '4', '5', '6', '6', '6'], ['1', '1', '1', '2', '0', '1', '1', '1', '2', '0'], [0.5, 0.2, 0.2, 0.5, 0.5, 0.5, 0.2, 0.5, 0.5, 0.5]);
```

Comment:
Would rather have this in a local README.md file that you can copy and paste from whenever you need to test this. As long as we don't have proper unit tests it's better not to have test code.